### ISSUE TITLE
Get center of map when user drag

### ISSUE THREAD

**Issue by oscarrock** [hansemannn/ti.googlemaps#38]
I am trying to get center coordinates of map when user drag the map view. I have succesfully get center of map on first load with regionchanged event.

```
mapView.addEventListener("regionchanged", handleCameraChangeEvent);
function handleCameraChangeEvent(e) {
    var coord = {latitude: e.map.region.latitude, longitude: e.map.region.longitude };
    drawMarker(coord);
}

```

But when user drag the map view that e.map.region position don't change. Any clues?

Thanks for this module, it's really helpfull i don't like apple maps.

**Comment by yozef** [comment]
it's in my pull request, however I leave today on vacation so won't have time to work more on it. `regionchanged` is suppose to return you the long / lat. However the current version needs to be fixed, which is part of the PR.

**Comment by oscarrock** [comment]
Ok budy, thanks. Awesome work.

**Comment by hansemannn** [comment]
Just merged #37 that includes the fix, thanks!
